### ISSUE TITLE
[v0.38] `ToggleFocusFullscreen` not working with `Run` anymore

### ISSUE THREAD

**Issue by utkarshgupta137** [zellij-org/zellij#2752]
**Basic information**
OS:
`zellij --version`: `0.38` (latest) 
`stty size`: `65 240`
`uname -av` or `ver`(Windows): `Darwin utkarsh-mbp 22.5.0 Darwin Kernel Version 22.5.0: Thu Jun  8 22:22:20 PDT 2023; root:xnu-8796.121.3~7/RELEASE_ARM64_T6000 arm64`

CLI:
`nvim --version`: '`NVIM v0.9.1`
`lazygit --version`: `commit=, build date=, build source=nix, version=0.40.2, os=darwin, arch=arm64, git version=2.39.2 (Apple Git-143)`
`alacritty --version`: `alacritty 0.12.2`

**Further information**
Up until v0.37.2, the following binding:
```kdl
    shared_except "locked" {
        bind "Alt g" {
            Run "lazygit" {
                close_on_exit true
            };
            ToggleFocusFullscreen;
        }
    }
```
Would result in lazygit opening in fullscreen. Now, it looks like the `ToggleFocusFullscreen` has no effect.

Weirdly, if I use the following binding after lazygit is opened:
```kdl
    pane {
        bind "f" { ToggleFocusFullscreen; SwitchToMode "Normal"; }
    }
```
Then it makes it fullscreen.

**Comment by imsnif** [comment]
Good catch!! Thanks for the quick report. I issued a fix, and will release a patch version with this and other stuff later today.

**Comment by utkarshgupta137** [comment]
Thanks for the quick turnaround <3
